test/index-feeds.js

```diff
@@ -0,0 +1,68 @@
+const test = require('tape');
+const path = require('path');
+const ssbKeys = require('ssb-keys');
+const SecretStack = require('secret-stack');
+const {where, type, count, toPromise} = require('ssb-db2/operators');
+const {generateAndTest, db2MigrationDone} = require('./utils');
+
+test('generation supports simulating meta feeds and index feeds', (t) => {
+  const M = 50;
+  const A = 8;
+  const I = 100;
+
+  generateAndTest(
+    {
+      outputDir: 'ssb-fixtures-test-index-feeds',
+      seed: 'banana',
+      messages: M,
+      authors: A,
+      indexFeeds: I,
+    },
+    async (err, msgs, cleanup, outputDir) => {
+      // (seed + announce + add main + add indexes + add 5 index) * A
+      const META_MSG_COUNT = 9 * A;
+      // Seems like we should have only `M` index feed msgs, right?
```

Comment:
Good call including these clarifying comments!

Reply:
This looks simple but it was torturing my mind to discover it :D

Reply:
Haha I could tell it must have been a mind-bender if you took the time to document it like this :D Well done.